media/css/privacy/firefox-privacy.scss

```diff
@@ -0,0 +1,282 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import
+    '../pebbles/includes/lib',
+    '../pebbles/includes/fonts/open-sans',
+    '../pebbles/includes/mixins/typography';
+
+* {
+    @include open-sans;
+}
+
+@font-face {
+    font-family: 'Open Sans';
+    font-weight: 600;
+    font-style: normal;
+    src: url('/media/fonts/opensans-semibold.woff2') format('woff2'),
+        url('/media/fonts/opensans-semibold.woff') format('woff');
+}
+
+$light-gray-background: #f2f2f2;
+
+.privacy-head,
+.privacy-lead-in,
+.privacy-footnote {
+    background-color: $light-gray-background;
+    color: #333;
+}
+
+/* POLICY HEADER */
+.privacy-head {
+    padding: 20px;
+    text-align: left;
+
+    @media #{$mq-desktop} {
+        position: sticky;
+        top: 0;
+        z-index: 2;
+
+        .privacy-head-wrapper {
+            margin: 0 auto;
+            padding-left: 17px;
+            max-width: 1420px;
+        }
+    }
+
+    h2 {
+        display: inline-block;
+        @include font-size-level3
+
+        @media #{$mq-desktop} {
+            margin: 0;
+        }
+
+        a {
+            color: inherit;
+            text-decoration: none;
+        }
+
+        &:before {
+            display: inline-block;
+            content: '';
+            @include at2x('/media/img/firefox/australis/fx38.0.5/firefox-logo.png', 30px, 33px);
+            width: 30px;
+            height: 33px;
+            vertical-align: middle;
+
+            @media #{$mq-tablet} {
+                background-size: 40px 43px;
+                width: 40px;
+                height: 43px;
+            }
+
+            @media #{$mq-desktop} {
+                background-size: 60px 63px;
+                width: 60px;
+                height: 63px;
+            }
+        }
+    }
+
+    .privacy-header-firefox {
+        display: none;
+
+        @media #{$mq-desktop} {
+            display: inline-block;
+            margin-left: 10px;
+        }
+    }
+
+    .privacy-header-policy {
+        font-weight: normal;
+    }
+
+    time {
+        display: inline-block;
+        @include font-size-level6
+        font-weight: normal;
+        font-style: italic;
+    }
+}
+
+/* POLICY LEAD-IN */
+.privacy-lead-in {
+    padding: $gutter-width 0 0;
+
+    h2 {
+        margin-bottom: 20px;
+        @include font-size-level3
+        font-weight: 600;
+        text-align: left;
+
+        @media #{$mq-tablet} {
+            font-weight: normal;
+        }
+
+        @media #{$mq-desktop} {
+            margin-bottom: (20px * 2);
```

Comment:
Nit: simplify and use `40px` (same for other occurences below)

Reply:
Remnant from when I was using `$gutter-width` ;)